test-cases/longevity/longevity-cdc-100gb-4h.yaml

```diff
@@ -1,11 +1,13 @@
 test_duration: 290
 
+pre_create_keyspace: "CREATE KEYSPACE cdc_test WITH replication = {'class': 'NetworkTopologyStrategy', 'eu-westscylla_node_west': '3'}  AND durable_writes = true;"
```

Comment:
@soyacz / @fruch another problem I see now is that we're going to be tightly coupled with the "region" mentioned here.
It means that if someone runs the test from a different region (can be changed from pipeline params), the test won't work.

Reply:
@roydahan I agree, this problem exists currently anyway and I already created some task regarding this: https://github.com/scylladb/scylla-cluster-tests/issues/5969